nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -567,6 +575,11 @@ Returns a faction datum by its name (case insensitive!)
 		COMSIG_ATOM_ENTERED = PROC_REF(on_entered),
 	)
 	AddElement(/datum/element/connect_loc, loc_connections)
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/effect/overmap_anomaly/LateInitialize()
+	. = ..()
+	GLOB.overmap_anomalies += src
```

Comment:
Can't this just be done in the regular initialize?

Reply:
It can, I just mirrored what we used to do for overmap ships.